components/ConfirmationModal.tsx

```
@@ -102,28 +104,31 @@ const ConfirmationModal = ({
-            onClick={onClose}
             data-testid="modal-backdrop"
+            exit={{ opacity: 0 }}
+            initial={{ opacity: 0 }}
           />
 
           {/* Modal */}
-          <div className="fixed inset-0 z-50 flex items-center justify-center p-4">
+          <div
+            className="fixed inset-0 z-50 flex items-center justify-center p-4"
+            onClick={onClose}
+          >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)